src/Symfony/Bundle/FrameworkBundle/Test/BrowserKitAssertionsTrait.php

```diff
@@ -47,7 +47,13 @@ public static function assertResponseRedirects(string $expectedLocation = null,
     {
         $constraint = new ResponseConstraint\ResponseIsRedirected();
         if ($expectedLocation) {
-            $constraint = LogicalAnd::fromConstraints($constraint, new ResponseConstraint\ResponseHeaderSame('Location', $expectedLocation));
+            if (class_exists(ResponseConstraint\ResponseHeaderLocationSame::class)) {
+                $locationConstraint = new ResponseConstraint\ResponseHeaderLocationSame(self::getRequest(), $expectedLocation);
```

Comment:
Would it make sense to have a more generic `ResponseHeaderUrlSame` for all headers consisting of a URL?

Reply:
I'm not sure. The behavior for `Location` depends on the URL of the request, would this behavior make sense for any kind of URL?